pokemongo_bot/__init__.py

```diff
@@ -101,7 +101,9 @@ def update_web_location(self, cells=[], lat=None, lng=None, alt=None):
                                                      gym_latitude=fort.get('latitude'),
                                                      gym_longitude=fort.get('longitude'))
                             response_gym_details = self.api.call()
-                            fort['gym_details'] = response_gym_details['responses']['GET_GYM_DETAILS']
+                            gym_details = response_gym_details.get('responses', {}).get('GET_GYM_DETAILS', None)
```

Comment:
you need need one line to fix this bug: `fort['gym_details'] = response_gym_details.get('responses', {}).get('GET_GYM_DETAILS', None)`